build.sh

```diff
@@ -55,7 +55,7 @@ mkdir vendor-prefixed
 ./tools/vendor/composer.phar install --no-dev --prefer-dist --optimize-autoloader --no-scripts
 
 echo '[BUILD] Fetching prefixed production libraries'
-./tools/vendor/composer.phar install --prefer-dist --working-dir=./prefixer/
+./tools/vendor/composer.phar install --no-dev --prefer-dist --working-dir=./prefixer/
```

Comment:
Could you explain this change to me? This is different than what we do in the ci and locally, right? So we will release a different version than we test. Could we break the production build by this?

Reply:
With this flag composer skips required-dev, dependencies. We do the same for non-prefixed dependencies. Take a look  a couple of lines above. We don't have any dev dependencies in prefixed composer.json so actually, this change will not affect anything. It only allows us adding prefixed dev dependencies, that won't be added to build.
I added this change because I needed to add dev dependency symfony/config, but I removed it later and used a workaround. I think we can keep this change it makes no harm.